app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_status_next_transition_validator.py

```diff
@@ -0,0 +1,184 @@
+from django.test import TestCase
+from parameterized import parameterized
+
+from experimenter.base.models import SiteFlag, SiteFlagNameChoices
+from experimenter.experiments.api.v5.serializers import NimbusExperimentSerializer
+from experimenter.experiments.models import NimbusExperiment
+from experimenter.experiments.tests.factories import NimbusExperimentFactory
+from experimenter.openidc.tests.factories import UserFactory
+
+
+class TestNimbusStatusNextTransitionValidator(TestCase):
+    maxDiff = None
+
+    def setUp(self):
+        super().setUp()
+        self.user = UserFactory()
+
+    def test_launch_request_while_disabled_error(self):
+        SiteFlag(name=SiteFlagNameChoices.LAUNCHING_DISABLED.name, value=True).save()
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED
+        )
+        serializer = NimbusExperimentSerializer(
+            experiment,
+            data={
+                "status": NimbusExperiment.Status.DRAFT,
+                "status_next": NimbusExperiment.Status.LIVE,
+                "publish_status": NimbusExperiment.PublishStatus.REVIEW,
+                "changelog_message": "Review Requested for Launch",
+            },
+            context={"user": self.user},
+        )
+        self.assertFalse(serializer.is_valid())
+        self.assertEqual(
+            serializer.errors["status_next"][0],
+            NimbusExperiment.ERROR_LAUNCHING_DISABLED,
+        )
+
+    @parameterized.expand(
+        [
+            [
+                NimbusExperiment.Status.DRAFT,
+                True,
+            ],
+            [
+                NimbusExperiment.Status.PREVIEW,
+                True,
+            ],
+            [
+                NimbusExperiment.Status.LIVE,
```

Comment:
is it possible `status as Live` and `status_next` as live too if its an experiment????